tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -860,7 +860,7 @@ shared_ptr<DuckDBPyConnection> DuckDBPyConnection::Rollback() {
 	return shared_from_this();
 }
 
-py::object DuckDBPyConnection::GetDescription() {
+Optional<py::list> DuckDBPyConnection::GetDescription() {
```

Comment:
I'm kind of confused by this being a list in general, but I guess that's out of the scope of this PR

Reply:
Why is this confusing? It's a list of tuples that contain the column metadata.

So
```py
[
   ('column0', 'bool', None, None, ...),
   ...
]
``|

Reply:
Ah sure, makes sense then, just the name description makes me think string
Again, out of scope :+1:

Reply:
Yeah the name is from the python dbapi standard